pkg/tfvars/nutanix/nutanix.go

```diff
@@ -0,0 +1,68 @@
+package nutanix
+
+import (
+	"encoding/json"
+
+	nutanixapis "github.com/openshift/machine-api-provider-nutanix/pkg/apis/nutanixprovider/v1beta1"
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/tfvars/internal/cache"
+	nutanixtypes "github.com/openshift/installer/pkg/types/nutanix"
+)
+
+type config struct {
+	PrismCentralAddress    string `json:"prism_central_address"`
+	Port                   string `json:"port"`
+	Username               string `json:"username"`
+	Password               string `json:"password"`
```

Comment:
The json name of these fields should start with "nutanix_".

Reply:
Fixed.